_posts/0000-01-02-morganwills.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
New item
- [ ] I didn't know this is how you create check boxes in markdown!
![giphy](https://user-images.githubusercontent.com/12152549/164571571-3a2dbad1-ce81-4fb5-bda4-fbfb59d1e47b.gif)